### ISSUE TITLE
TOC sidebar does not stay on the screen

### ISSUE THREAD

**Issue by htfv** [andriusvelykis/reflow-maven-skin#2]
If the TOC is placed in the sidebar (`<toc>sidebar</toc>`) and I scroll a long page down, it is scrolled with the page.

It think it would be better if sidebar jumps to the fixed position when the page is scrolled down.

**Comment by andriusvelykis** [comment]
That is by design. I chose to heave sidebar ToC scroll in a long page instead of jumping to a fixed position.

The rationale is that if you have a small screen (e.g. 11'' laptop) and the sidebar ToC is long, the bottom of the ToC is not accessible at all (it hides below the bottom of the screen and you cannot scroll to it, since it is fixed).

I suppose this could become a configuration option, e.g. force fixing if you want or can ensure that the ToC is small enough. I'll leave this as a feature suggestion. Alternatively, you could try adding custom CSS to your code to achieve that if this is needed immediately.

**Comment by htfv** [comment]
Yes, I understood the point right after I submitted the issue. May be a configuration option and/or a floating button "to the top"?
